cdc/owner.go

```diff
@@ -784,7 +791,7 @@ func (o *Owner) flushChangeFeedInfos(ctx context.Context) error {
 		// If a stopped changefeed's CheckpoinTs < minGCSafePoint, means this changefeed is stagnant.
 		// It should never be resumed. This part of the logic is in newChangeFeed()
 		// So here we can skip it.
-		if status.CheckpointTs < minGCSafePoint {
+		if minGCSafePoint != 0 && status.CheckpointTs <= minGCSafePoint {
```

Comment:
ditto

Reply:
- We need the `<=` check here is because when a `changefeed` is stagnant, its `checkpointTs` will be updated to `pd`, and it would be the minimum `gcSafePoint` of `cdc`. In order to avoid circular dependence, it is necessary to check for equality.  
- Checking whether they are equal will make the loop assignment to happen only once, and then it will be skipped in next time.
- The situation here is a bit complicated. Since it is not possible to get `tikv_gc_life_time` directly from pd, I can only take this dirty hack.

Reply:
ditto

Reply:
done